dind/node/Dockerfile

```diff
@@ -41,7 +41,11 @@ RUN mkdir -p /opt/cni/bin
 RUN dpkg -i kubectl.deb
 RUN dpkg -i kubernetes-cni.deb
 RUN dpkg -i kubelet.deb
-RUN dpkg -i kubeadm.deb
+ENV CRI_VERSION="v1.11.1"
+RUN curl -L https://github.com/kubernetes-incubator/cri-tools/releases/download/$CRI_VERSION/crictl-$CRI_VERSION-linux-amd64.tar.gz | \
+    tar zxvf - -C /usr/local/bin
+# force due to dependency on missing cri deb for now
+RUN dpkg -i --force depends kubeadm.deb
```

Comment:
there's a CRI deb in k/k build we can consume https://github.com/kubernetes/kubernetes/commit/03456a33f9cd1123572a4a20e3728b65d5fb6925

Reply:
Updated to include! Thanks for helping me find that!